app/models/Topic.scala

```diff
@@ -158,9 +158,9 @@ object Topic {
     }
   }
 
-  def deleteUnlinkedBefore(date: Date) = {
+  def deleteUnlinkedBefore(date: Date) {
     DB.withConnection { implicit c =>
-      SQL("delete from topic where created < {created}").on('created -> date)
+      SQL("delete from topic where created < {created}").on('created -> date).executeUpdate
```

Comment:
I enabled this query to execute. It'll still runtime error if called on Topics with linked Items due to foreign key restrictions.

See https://github.com/MITLibraries/scoap3hub/issues/141